pom.xml

```diff
@@ -67,7 +66,27 @@
     <dependency>
       <groupId>io.rest-assured</groupId>
       <artifactId>rest-assured</artifactId>
-      <version>3.0.2</version>
+      <version>3.3.0</version>
+      <scope>test</scope>
+      <exclusions>
+        <exclusion>
+          <groupId>com.sun.xml.bind</groupId>
+          <artifactId>jaxb-osgi</artifactId>
+        </exclusion>
+      </exclusions>
+    </dependency>
+    <dependency>
+      <!-- forcing 2.3.0 in dep for rest-assured to avoid systemPath issue on JDK11,
+      see https://github.com/jenkinsci/jaxb-plugin/pull/6 for more information -->
+      <groupId>javax.xml.bind</groupId>
+      <artifactId>jaxb-api</artifactId>
+      <version>2.3.0</version>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.apache.commons</groupId> <!-- forced bc RequireUpperBoundDeps -->
+      <artifactId>commons-lang3</artifactId>
+      <version>3.7</version>
```

Comment:
It overrides the version from the Jenkins core IIUC. In such case you would not achieve much by forcing it here, because the older version from the core maybe classloaded before the plugin starts. Better to add an exclusion and a TODO comment with a JIRA ticket.

Reply:
Given this is a test scope only dependency, I am wondering if this really matters that much, but fine by me. Doing.

Reply:
On `master` branch currently (`f7d6d921fbfd34cf336759e7ac1598b158892a5e`):

```
mvn dependency:tree -Dincludes=":commons-lang3"                                                                                                                master
[INFO] Scanning for projects...
[INFO]
[INFO] ---------------< org.jenkins-ci.plugins:display-url-api >---------------
[INFO] Building Display URL API 2.3.1-SNAPSHOT
[INFO] --------------------------------[ hpi ]---------------------------------
[INFO]
[INFO] --- maven-dependency-plugin:3.0.0:tree (default-cli) @ display-url-api ---
[INFO] org.jenkins-ci.plugins:display-url-api:hpi:2.3.1-SNAPSHOT
[INFO] \- io.rest-assured:rest-assured:jar:3.0.2:test
[INFO]    \- io.rest-assured:xml-path:jar:3.0.2:test
[INFO]       \- org.apache.commons:commons-lang3:jar:3.4:test
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  2.661 s
[INFO] Finished at: 2019-03-22T09:42:22+01:00
[INFO] ------------------------------------------------------------------------
```

Reply:
Done